internal/app/app.go

```diff
@@ -79,11 +79,18 @@ type FileChange struct {
 	Patch       string `json:"patch"`
```

Comment:
Adding documentation for `PRReviewMap` is useful for maintaining the codebase. However, ensure that the comment explicitly describes the purpose and usage of the structure for better clarity.